modules/reference/pages/node-properties.adoc

```diff
@@ -116,14 +116,14 @@ Flag to enable developer mode, which skips most of the checks performed at start
 
 === empty_seed_starts_cluster
 
-Controls how a new cluster is formed. This property must have the same value in all brokers in a cluster.
+Controls how a new cluster is formed. All brokers in a cluster must have the same value.
 
-If `true`, an empty <<seed_servers,seed_servers>> list denotes that this broker should form a cluster. At most, one broker in the cluster should be configured with an empty seed_servers list. If no such configured broker exists, or if configured to be `false`, then all brokers denoted by the seed_servers list must be identical in their configurations, and those brokers form the initial cluster.
-
-TIP: This is set to `true` by default for backward compatibility, but Redpanda recommends setting `empty_seed_starts_cluster` to `false`.
+<<seed_servers,See how the empty_seed_starts_cluster setting works with the seed_servers setting>> to form a cluster.
 
 *Default*: true
 
+TIP: For backward compatibility, `true` is the default. Redpanda recommends using `false`.
```

Comment:
```suggestion
TIP: For backward compatibility, `true` is the default. Redpanda Data recommends using `false`.
```

Reply:
It would be good to say why we recommend `false`.

Reply:
I think [this code](https://github.com/redpanda-data/redpanda/blob/465118e4672c15bab1e19c86d4f886cd74e95742/src/v/config/node_config.cc#L68) still confirms that the default is `true`, but there were no clues via the code comments that explicitly say why `false` is recommended. My hunch is that it avoids unexpected/accidental cluster formation and provides clarity via explicit configuration. Would need to verify from someone more knowledgeable.

Reply:
@BenPope

Reply:
@BenPope @mmaslankaprv could you confirm?

Reply:
> My hunch is that it avoids unexpected/accidental cluster formation and provides clarity via explicit configuration.

I believe this is the case. @michael-redpanda ?

Reply:
Indeed it prevents accidental cluster formation. It should be recommended to `false` in production deployments.